src/components/Banner/Banner.tsx

```diff
@@ -9,23 +9,13 @@ import {
   CircleInformationMajorTwotone,
```

Comment:
This change is only removing `withAppProvider` since it's not being used and using `WithinContentContext` consumer rather than `withContext`